.npmignore

```diff
@@ -0,0 +1,11 @@
+__tests__
+.vscode
```

Comment:
From my point of view, it's better to remove this file from repo completely, if it doesn't contain any specific settings, which are necessary for working on a project.

Reply:
.vscode is added to .gitignore and need to be ignored here as well